src/stories/Library/Heros/hero/HeroInner.tsx

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+import Tag from "../../tag/Tag";
+import ListDescription, {
+  ListData,
+} from "../../Lists/list-description/ListDescription";
+
+export type HeroInnerProps = {
+  contentType?: string;
+  date?: string;
+  title: string;
+  description?: string;
+  descriptionItems?: ListData;
+  cta?: string;
+  tag?: string;
+};
+
+const HeroInner: React.FunctionComponent<HeroInnerProps> = ({
+  contentType,
+  date,
+  title,
+  description,
+  descriptionItems,
+  cta,
+  tag,
+}) => {
+  return (
+    <div className="hero__inner">
+      {tag ? (
```

Comment:
Here you can use the && operator

Example:
function WelcomeMessage() {
  const [isLoggedIn, setIsLoggedIn] = React.useState(false);

  return (
    <div>
      {isLoggedIn && <h1>Velkommen tilbage!</h1>}
    </div>
  );
}